shared/test/integration/helpers/marionetteSearchTypes.js

```diff
@@ -0,0 +1,18 @@
+
+'use strict';
+
+function MarionetteSearchTypes() {}
+
+MarionetteSearchTypes.prototype = {
+    cssSelector : 'css selector',
+    xpath : 'xpath',
+    cssClass : 'class name',
+    name : 'name',
+    cssId : 'id',
+    tagName : 'tag name',
+    linkText : 'link text',
+    partialLinkText : ' partial link text'
```

Comment:
Extra space at the beginning of the string. It might be correct, but it looks odd. If that's a correct one, I'd add a comment to explain the reason